book-store-spring/src/main/java/mate/academy/bookstorespring/model/Book.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.bookstorespring.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
```suggestion
@Getter
@Setter
@Table(name = "books")
```

Annotation Data could cause some issues, try to avoid using it for Entities